platform/darwin/src/MGLSignpost.h

```diff
@@ -0,0 +1,72 @@
+#ifndef MGLSignpost_h
+#define MGLSignpost_h
+
+#include <os/log.h>
+#include <os/signpost.h>
+
+#define SIGNPOST_CONCAT2(x,y)   x##y
+#define SIGNPOST_CONCAT(x,y)    SIGNPOST_CONCAT2(x,y)
+#define SIGNPOST_NAME(x)        SIGNPOST_CONCAT(signpost,x)
+
+#define MGL_EXPORT __attribute__((visibility ("default")))
```

Comment:
Addressed in comment https://github.com/mapbox/mapbox-gl-native/pull/15203#issuecomment-515647740 - perhaps it would be better NOT to define `MGL_EXPORT` in this case.